infrastructure/main.tf

```diff
@@ -23,6 +23,7 @@ locals {
     // configures a user for a jurisdiction
     // add secrets to all bulk-scan vaults in the form idam-users-<jurisdiction>-username idam-users-<jurisdiction>-password
     SSCS = "idam-users-sscs"
```

Comment:
Do we need this ?

Reply:
Excel spreadsheet updated. I've been told by @timja we do need that.

Reply:
well, no point removing it.
the creds are there for SSCS, and we're about to spike out integrating with them